src/test/java/teammates/test/cases/ui/AdminSearchPageActionTest.java

```diff
@@ -12,8 +12,8 @@ public static void classSetUp() {
     
     @Test
     public void testExecuteAndPostProcess() {
-        // TODO : Test is done as a browser test,
-        // cuz gaeSimulation gives problems when rebuilding the document
+        // Test is done as a browser test,
+	// because otherwise there are problems when rebuilding the document
```

Comment:
Shift this line one space to the right, so that the comments are in line with each other.